apps/dashboard-e2e/src/describer/modelAssessment/modelOverview/describeAxisConfigDialog.ts

```diff
@@ -0,0 +1,67 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+export function describeAxisConfigDialog(): void {
```

Comment:
+1 on this

Reply:
Because the chart here will change, and so will the tests. I don't think we'll actually change the interpret charts. If you refer to my document, you'll find that there will be a range of different chart options.